tests/dispatch/signature/test_signature.py

```diff
@@ -146,58 +146,58 @@ def test_known_signature(self):
     @mock.patch.dict(os.environ, {"DISPATCH_VERIFICATION_KEY": public_key2_pem})
     def test_parse_verification_key_env_pem_str(self):
         verification_key = parse_verification_key(None)
-        self.assertIsInstance(verification_key, Ed25519PublicKey)
-        self.assertEqual(verification_key.public_bytes_raw(), public_key2_bytes)
+        assert isinstance(verification_key, Ed25519PublicKey)
+        assert verification_key.public_bytes_raw(), public_key2_bytes
```

Comment:
Using `assert` instead of the methods helped mypy understand that `verification_key` couldn't be `None`.